extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -100,6 +100,7 @@ import {
 import {
   addExperimentIdToElement,
 } from '../../../ads/google/a4a/traffic-experiments';
+import {getAdCid} from '../../../src/ad-cid';
```

Comment:
I don't see you using this yet.  Presumably you'll remove and handle in second PR?